src/emfcodec.c

```diff
@@ -266,7 +266,6 @@ PolyPolygon (MetafilePlayContext *context, BYTE *data, BOOL compact)
 	n++;
 
 	/* total number of points (in all polygons)*/
-	int total = GETDW(DWP(n));
```

Comment:
this might be helpful for debugging and the comment above would be confusing then. Please just comment it out instead.

Reply:
fixed